atlp-rwanda/wolves-bn-backend
# Ft search functionality #174772231

> #### What does this PR do?
- it adds a search for destination, departure, and owners field engine
> #### Description of Task to be completed
- add a search controller
- test search endpoint
> #### How should this be manually tested?
- git clone the application URL(`https://github.com/atlp-rwanda/wolves-bn-backend.git`)
- run `npm install`
- run `npm start` and access endpoints `api/trips/search?{query}` and `api/trips/search?{query}`
> #### What are the relevant Pivotal tracker stories?
-https://www.pivotaltracker.com/story/show/174772231
>### screenshots
![search](https://user-images.githubusercontent.com/49249449/95173249-b52e0100-07b8-11eb-997b-f6c6f0367a04.PNG)
